src/main/kotlin/org/jetbrains/plugins/bsp/magicmetamodel/impl/workspacemodel/GoWorkspaceModelEntities.kt

```diff
@@ -16,6 +17,7 @@ public data class GoModule(
   val importPath: String,
   val root: Path,
   val goDependencies: List<GoModuleDependency>,
+  val goLibraries: List<LibraryItem> = emptyList(),
```

Comment:
to tak samo trzeba dodać tam niżej do funkcji toState

Reply:
Żeby przetestować te wszystkie toState, to niestety trzeba się trochę nawysilać, w skrócie jak po zindeksowaniu projektu zamyka się IntelliJ, to on w którymś miejscu tworzy plik magicmetamodel.xml, z którego będzie czytał przy pierwszym otworzeniu projektu po ponowny odpaleniu IntelliJ

te wszystkie State powinny mieć pola tylko typu String, Boolean albo inne State (no i jakieś inne bardzo proste ale to wyjątki) - bo w skrócie State chcemy w łatwy sposób zapisać potem w pliku i z niego odczytać, tak żeby się dało przywrócić do tego co było